core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/version/Version.java

```diff
@@ -172,6 +172,7 @@ public String getResourceId() {
 		return getResource().getResourceId();
 	}
 	
+	@JsonIgnore
```

Comment:
This is a valid API response property. Please remove this `JsonIgnore` annotation.